tests/mcproto/types/test_nbt.py

```diff
@@ -1246,7 +646,7 @@ def to_nbt(self, name: str = "") -> NBTag:
             {"test": object()},
             ByteNBT,
             TypeError,
-            r"Expected one of \(bytes, str, int, float, list\), but found object.",
+            "Expected one of \\(bytes, str, int, float, list\\), but found object.",
```

Comment:
What do you have against raw strings? They're perfectly fine, especially for regex patterns. This was a change I explicitly made after merging #257 in e05fd2d. Please revert.

Reply:
Oops sorry, I may have reverted this while copy/pasting stuff from a different branch during the merge.